superset_patchup/views.py

```diff
@@ -0,0 +1,61 @@
+"""
+Additional extensions to SuperSet views
+"""
+
+import json
+from flask_appbuilder import expose
+from flask_appbuilder.models.sqla.interface import SQLAInterface
+import superset
+
+from . import (VERSION, __version__)
+
+class SupersetKetchupApiView(superset.views.base.BaseSupersetView):
+
+    """
+    Extensions to Superset API
+    """
+
+    def __init__(self):
+        self.route_base = "/superset-ketchup/api"
+
+    @superset.views.base.api
+    @expose('/version/', methods=['GET'])
+    def version(self):
+        """Returns *superset-api* version"""
```

Comment:
Figured this would be nice to have, and can use in ansible-superset stuff too to test that our api is actually getting loaded

Reply:
what is the superset-api version in this case? superset or ketchup?

Reply:
whoops, bad comment.  superset-ketchup version